cli/azd/cmd/root.go

```diff
@@ -90,8 +90,6 @@ func NewRootCmd(
 		DisableAutoGenTag: true,
 	}
 
-	rootCmd.CompletionOptions.HiddenDefaultCmd = true
```

Comment:
Wondering why did we do this in the past....  Would the default command fail if there is not a definition for the command like the one you are adding?

Reply:
> Would the default command fail if there is not a definition for the command like the one you are adding?

There's a separate option `DisableDefaultCmd` for that:

<img width="1561" height="499" alt="image" src="https://github.com/user-attachments/assets/2ca57f55-f6c3-4bbc-9481-470500d431b0" />

I'm not sure why even after removing `HiddenDefaultCmd  = true`, it doesn't show in `azd --help`. Maybe because of how we override the default `--help` text in https://github.com/Azure/azure-dev/blob/69d00310ac3dcf9dc47d71f33aa3f87649a6d02a/cli/azd/cmd/root.go#L471-L473

In any case, the docs suggest this approach to provide a custom implementation: https://github.com/spf13/cobra/blob/main/site/content/completions/_index.md#creating-your-own-completion-command